examples/tv-casting-app/android/App/app/src/main/jni/cpp/TvCastingApp-JNI.cpp

```diff
@@ -153,7 +155,8 @@ JNI_METHOD(jboolean, initServer)(JNIEnv * env, jobject, jobject jCommissioningCo
     CHIP_ERROR err = SetUpMatterCallbackHandler(env, jCommissioningCompleteHandler, gCommissioningCompleteHandler);
     if (err == CHIP_NO_ERROR)
     {
-        CastingServer::GetInstance()->InitServer(CommissioningCompleteHandler);
+        // CHIP Server was already initialized for this Android app
+        CastingServer::GetInstance()->InitServer(CommissioningCompleteHandler, false);
         return true;
     }
     else
```

Comment:
please prefer no else after return. 

You may want to switch order: return early on failure.

Reply:
Done